libraries/standard/http/include/http_client.h

```diff
@@ -0,0 +1,339 @@
+#ifndef HTTP_CLIENT_H_
```

Comment:
No explicit license text needed per file?

Reply:
The license is not decided we can add that later.